Issue.py

```diff
@@ -2,7 +2,7 @@ class Issue(object):
     """Basic Issue model for collecting the necessary info to send to GitHub."""
 
     def __init__(self, title, labels, assignees, milestone, body, hunk, file_name,
-                 start_line, num_lines, markdown_language, status, identifier, ref, issue_url, issue_number):
+                 start_line, num_lines, markdown_language, status, identifier, identifier_actual, ref, issue_url, issue_number, start_line_within_hunk=1):
```

Comment:
`identifier_actual` is the _actual_ identifier that was matched, whereas `identifier` is the canonical _class_ of identifier that was matched. For example, for the following comment:
```
# todo: do the thing
```
`identifier_actual` is `todo` and `identifier` is `TODO`